projects/js-packages/components/components/button/index.jsx

```diff
@@ -37,7 +37,7 @@ const Button = ( {
 		[ styles.normal ]: size === BUTTON_SIZES.NORMAL,
 		[ styles.small ]: size === BUTTON_SIZES.SMALL,
 		[ styles.icon ]: Boolean( icon ),
-		propsClassName,
+		[ propsClassName ]: Boolean( propsClassName ),
```

Comment:
You could only move propsClassName as a new param.

`classNames( ..., {}, propsClassName)`

I thought I was doing that. 🤦‍♂️

Reply:
fd752e626f8e20edc41a6b29fce4fdfe36e5f45c